docs/Animations.md

```diff
@@ -444,14 +445,14 @@ var App = React.createClass({
 
     return (
       <View style={styles.container}>
-        <TouchableWithoutFeedback onPressIn={this._onPressIn}
-                                  onPressOut={this._onPressOut}>
+        <TouchableWithoutFeedback onPressIn={() => this._onPressIn()}
+                                  onPressOut={() => this._onPressOut()}>
```

Comment:
BTW the better approach is to bind them in the constructor to avoid new function creation on every render.